packages/terra-clinical-item-display/src/terra-dev-site/doc/clinical-item-display/clinicalItemDisplay.1.doc.mdx

```diff
@@ -60,14 +59,15 @@ Some examples of progressive disclosure patterns that may be used to disclose tr
 - Accordions
 - Dialogs
 - Popovers
-- Show/Hide
 - Split Views
 - Toasts
 
 The method of disclosure **must** be accessible via keyboard interactions.
 
-Truncation should be avoided where it is not necessary. Certain content should **never** be truncated (i.e. 
+Truncation should be avoided where it is not necessary. Certain content should **never** be truncated (i.e.
 medication names and dosages in menus where the user is selecting from a list of choices).
+Providing a shorter preview text with a method to disclose the full text provides a more equitable
+experience for all users.
```

Comment:
While this doesn't matter in the rendered output, the division of these lines seem arbitrary from a coding perspective. We can probably put one sentence  per line. e.g.
```suggestion
Truncation should be avoided where it is not necessary. 
Certain content should **never** be truncated (i.e. medication names and dosages in menus where the user is selecting from a list of choices).
Providing a shorter preview text with a method to disclose the full text provides a more equitable experience for all users.
```